deploy/helm/sumologic/values.yaml

```diff
@@ -592,10 +598,7 @@ prometheus-operator:
       interval:
   # Ensure we use pre 1.14 recording rules consistently as current content depends on them.
   kubeTargetVersionOverride: 1.13.0-0
-  ## Set the enabled flag to false for either of the below two purposes:
-  ## 1. Not install prometheus operator helm chart as a dependency along with this helm chart
-  ## 2. Disable metrics collection altogether
-  enabled: true
```

Comment:
Could we revert this removal?

Reply:
It has been moved up which I actually like.

Reply:
Could you link me the line?
I see only addition to logs and metrics

Reply:
added it back